### PR TITLE
Feature/b2 c 67 create account page ui

### PR DESCRIPTION
**Pull request by joseluisgraa**

### CODE REVIEW

File: b2c-app/src/components/Breadcrumbs/Breadcrumbs.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ function Breadcrumbs(props) {
         <div className="govuk-breadcrumbs">
             <ol className="govuk-breadcrumbs__list">
                 <li className="govuk-breadcrumbs__list-item">
```
Comment (DrogoNevets) [inline b2c-app/src/components/Breadcrumbs/Breadcrumbs.js:7]: should be a seperate breadcrumb component

File: b2c-app/src/components/Footer/Footer.js [hunk 2]
```diff
@@ -8,25 +12,11 @@ export default function footer() {
...
-                            Terms and conditions
-                            </a>
-                        </li>
-                    </ul>
+                    
+                    <components.FooterNavigation items={footerData.items} />
+                    
                     <div className="govuk-footer__meta-custom">
-                        Built by the <a href="./login" className="govuk-footer__link">Skills Funding Agency</a>
+                        Built by the <a href="https://www.gov.uk/government/organisations/skills-funding-agency" className="govuk-footer__link">Skills Funding Agency</a>
```
Comment (DrogoNevets) [inline b2c-app/src/components/Footer/Footer.js:19]: CHeck this with Sally, dont want another team getting credit for our work if we can avoid it

File: b2c-app/src/components/FooterNavigation/FooterNavigation.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+import components from '../../components';
+
+function FooterNavigation(props) {
+
+    //build array of components with actual config
+    const footerLinkComponents = props.items.map( 
+        item => <components.FooterLink title={item.title} link={item.link} key={item.title+item.link}/>
```
Comment (DrogoNevets) [inline b2c-app/src/components/FooterNavigation/FooterNavigation.js:9]: you know me better than this i like curly brackets so we can debug :P

File: b2c-app/src/components/HeaderTitle/HeaderTitle.js [hunk 4]
```diff
@@ -2,7 +2,7 @@ import React from 'react'
 
 function HeaderTitle(props) {
     return (
-        <a href="./signup" className="govuk-header__link govuk-header__link--service-name">
+        <a href="https://nationalcareers.service.gov.uk/" className="govuk-header__link govuk-header__link--service-name">
```
Comment (DrogoNevets) [inline b2c-app/src/components/HeaderTitle/HeaderTitle.js:5]: H tag?

File: b2c-app/src/components/Breadcrumbs/Breadcrumbs.js (hunk 1 above)
Comment (joseluisgraa) [reply b2c-app/src/components/Breadcrumbs/Breadcrumbs.js:7]: can we do it when we need them? for now it is always point to "home"

File: b2c-app/src/components/Footer/Footer.js (hunk 2 above)
Comment (joseluisgraa) [reply b2c-app/src/components/Footer/Footer.js:19]: Actually this is not on the live site so I'll remove it

File: b2c-app/src/components/FooterNavigation/FooterNavigation.js (hunk 3 above)
Comment (joseluisgraa) [reply b2c-app/src/components/FooterNavigation/FooterNavigation.js:9]: done

**Pull request merged**